README.md

```diff
@@ -11,30 +11,28 @@ This should work on any version of:
 - Puppet >= 2.7 that provides the `validate` face.
 - Ruby >= 1.8 with `erb` from stdlib.
 
-You can see the matrix of specific versions that we currently test against
-in the [TravisCI config](.travis.yml).
+You can see the matrix of specific versions that we currently test against in the [TravisCI config](.travis.yml).
 
-If you're using `puppetlabs_spec_helper/rake_tasks` and getting unexpected
-non-zero exit codes then you should upgrade to [puppetlabs_spec_helper][psh]
-\>= 0.8.0 which no longer has a conflicting rake task and now depends on
-this project.
+If you're using `puppetlabs_spec_helper/rake_tasks` and getting unexpected non-zero exit codes, upgrade to [puppetlabs_spec_helper][psh] version 0.8.0 or greater, which no longer has a conflicting rake task and now depends on this project. [TODO: This isn't totally clear to me. Puppet::Syntax depends on spec_helper 0.8.0 or greater, correct? Also, there is still a rake task, but it just no longer conflicts?]
```

Comment:
These days puppetlabs_spec_helper is integrating Puppet::Syntax correctly. Before 0.8.0, PSH was providing similar, but conflicting facilities. As a user, you either depend on, and integrate Puppet::Syntax yourself, or you use puppetlabs_spec_helper to do the same for you.